automatic/dropbox/tools/chocolateyinstall.ps1

```diff
@@ -1,18 +1,37 @@
-﻿$ErrorActionPreference  = 'Stop'
+$ErrorActionPreference  = 'Stop'
```

Comment:
If you removed BOM here, please add it back.
Powershell incorrectly reads script files as ANSI when no BOM is available.